themes/vocabulary_theme/templates/page-with-toc.html

```diff
@@ -61,53 +61,52 @@ <h1>{{ this.title }}</h1>
-                        ['/community/community-team/project-roles', 'Project Roles'],
-                        ['/community/community-team/community-building-roles', 'Community Building Roles'],
-                        ['/community/community-team/engagement-channels', 'Engagement Channels'],
-                      ] %}
-                        <li><a class="{% if this.path == href %} is-active {% endif%} link" href="{{ href|url }}"><i class="icon circle-filled"></i>{{ title }}</a></li>
-                      {% endfor %}
-                      </ul>
-                    {% endif %}
-                    {% if (href == '/community/code-of-conduct') %}
-                      <ul>
-                        <li><a class="{% if this._path == '/community/code-of-conduct/enforcement' %} is-active {% endif%} link" href="{{ '/community/code-of-conduct/enforcement'|url }}"><i class="icon circle-filled"></i>Code of Conduct Enforcement</a></li>
-                      </ul>
-                    {% endif %}
+                      <a class="{% if this.path == href %} is-active {% endif%} link" href="{{ href|url }}" aria-label="{{ title }}">{{ title }}</a>
+                      {% if (href == '/community/community-team') %}
+                        <ul>
+                          {% for href, title in [
+                            ['/community/community-team/members', 'Community Team Members'],
+                            ['/community/community-team/project-roles', 'Project Roles'],
+                            ['/community/community-team/community-building-roles', 'Community Building Roles'],
+                            ['/community/community-team/engagement-channels', 'Engagement Channels'],
+                          ] %}
+                            <li><a class="{% if this.path == href %} is-active {% endif%} link" href="{{ href|url }}" aria-label="{{ title }}"><i class="icon circle-filled"></i>{{ title }}</a></li>
+                          {% endfor %}
+                        </ul>
+                      {% endif %}
+                      {% if (href == '/community/code-of-conduct') %}
+                        <ul>
+                          <li><a class="{% if this._path == '/community/code-of-conduct/enforcement' %} is-active {% endif%} link" href="{{ '/community/code-of-conduct/enforcement'|url }}" aria-label="Code of Conduct Enforcement"><i class="icon circle-filled"></i>Code of Conduct Enforcement</a></li>
+                        </ul>
+                      {% endif %}
                     </li>
                     {% if (href != '/community/supporters') %}
                       <hr class="divider">
                     {% endif %}
                   {% endfor %}
-                  </li>
                 </ul>
               </aside>
             {% elif this.is_child_of('/programs') %}
-              <aside class="menu sidebar-menu">
+              <aside class="menu sidebar-menu" role="complementary" aria-label="Programs Sidebar">
```

Comment:
`aside` already has a semantic role, unsure this is necessary. Additionally if it needs a label it should likely have a heading within it to support that.